eng/emitters/pipelines/templates/steps/test-step.yml

```diff
@@ -50,3 +57,13 @@ steps:
       filePath: $(selfRepositoryPath)${{ parameters.PackagePath }}/eng/scripts/Test-Packages.ps1
       arguments: ${{ parameters.TestArgs }}
       workingDirectory: $(selfRepositoryPath)
+
+  - task: AzureCLI@2
```

Comment:
If you want to let it take effect right now, we should remove the task from autorest.csharp pipeline, otherwise the coverage from autorest.csharp will overwrite this one.

Reply:
This was removed here https://github.com/Azure/autorest.csharp/pull/4937